src/fix_wall_reflect.cpp

```diff
@@ -86,26 +86,26 @@ FixWallReflect::FixWallReflect(LAMMPS *lmp, int narg, char **arg) :
       iarg += 2;
 
     } else if (strcmp(arg[iarg],"units") == 0) {
-      if (iarg+2 > narg) error->all(FLERR,"Illegal wall/reflect command");
+      if (iarg+2 > narg) utils::missing_cmd_args(FLERR, "fix wall/reflect units", error);
       if (strcmp(arg[iarg+1],"box") == 0) scaleflag = 0;
       else if (strcmp(arg[iarg+1],"lattice") == 0) scaleflag = 1;
-      else error->all(FLERR,"Illegal fix wall/reflect command");
+      else error->all(FLERR,"Unknown fix wall/reflect units argument: {}", arg[iarg+1]);
       iarg += 2;
 
-    } else error->all(FLERR,"Illegal fix wall/reflect command");
+    } else error->all(FLERR,"Unknown fix wall/reflect keyword: {}", arg[iarg]);
   }
 
   // error check
 
-  if (nwall == 0) error->all(FLERR,"Illegal fix wall command");
+  if (nwall == 0) utils::missing_cmd_args(FLERR, "fix wall/reflect", error);
 
   for (int m = 0; m < nwall; m++) {
     if ((wallwhich[m] == XLO || wallwhich[m] == XHI) && domain->xperiodic)
-      error->all(FLERR,"Cannot use fix wall/reflect in periodic dimension");
+      error->all(FLERR,"Cannot use fix wall/reflect in xperiodic dimension");
```

Comment:
```suggestion
      error->all(FLERR,"Cannot use fix wall/reflect in periodic dimension x");
```
and equivalent below.